File: ios/GlassesEvents/GlassesEvents/Glasses/GlassesEventsScreen.swift
//Copyright (c) 2020 Everysight LTD

import Foundation
import NativeEvsKit

class GlassesEventsScreen : Screen {
    
    override func onCreate() {
        let text = Text()
        text.setText(text: "Glasses Events")
            .setResource(font: Font(stockFont: .medium))
            .setTextAlign(align: Align.center)
            .setX(x: getWidth()/2)
            .setY(y: getHeight()/2)
            .setForegroundColor(color: EvsColor.green.rgba)
        add(uiElement: text)
    }
}

File: android/GlassesEvents/src/main/java/com/everysight/sdk/samples/glassesevents/GlassesEventsScreen.kt
package com.everysight.sdk.samples.glassesevents

import UIKit.app.Screen
import UIKit.app.data.Align
import UIKit.app.data.EvsColor
import UIKit.app.resources.Font
import UIKit.widgets.Text


class GlassesEventsScreen:Screen() {
    internal val text = Text()
    override fun onCreate() {

        text.setText("Glasses Events").setResource(Font.StockFont.Medium).setTextAlign(Align.center)
        text.setX(getWidth()/2).setY(getHeight()/2).setForegroundColor(EvsColor.Green.rgba)
        add(text)
    }
}